addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzDiscoveryService.java

```diff
@@ -104,14 +105,15 @@ protected synchronized void stopScan() {
      * @param device Device model received from a FRITZ!Box
      */
     public void onDeviceAddedInternal(DeviceModel device) {
-        ThingUID thingUID = this.bridgeHandler.getThingUID(device);
+        final ThingUID thingUID = bridgeHandler.getThingUID(device);
```

Comment:
I agree. We recently had a discussion at ESH about it and decided that the recommendation is to use final mainly on fields, but not on local variables or parameters, so that the code does not get cluttered.